oat-sa/tao-core
# Fix/actp 283/bundling issue under windows

**Warning** Do not merge this PR before the companion PR. Once the companion is merged and released, update the version in the file `views/build/package.json` and run `npm i`. Then you'll can merge it.

Edit: companion PR has been merged, so you may proceed with the review and merge of this PR.

Related to: https://oat-sa.atlassian.net/browse/ACTP-283

Requires: 
 - [x] https://github.com/oat-sa/grunt-tao-bundle/pull/12

When developing under Windows, we cannot bundle the assets since the bundler task is raising the error:
```
>> Error: SyntaxError: Invalid Unicode escape sequence
>>     at C:\xampp\apps\tao\tao\views\build\node_modules\requirejs\bin\r.js:27738:40
Fatal error: Unable to bundle your code
```

How to test:
- run `npm i` in the `tao/views/build` folder
- under a *nix machine, make sure the bundling is working well by running a bundling task
- under a windows machine, with the TAO dev stack, run a bundling task.

example:
```
cd tao/views/build
npx grunt taobundle
```
